tensorboard/defs/resource_digest_suffixer_test.py

```diff
@@ -0,0 +1,117 @@
+# Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Tests for tensorboard.defs.resource_digest_suffixer."""
+
+import os
+import shutil
+
+from tensorboard import test as tb_test
+from tensorboard.defs import resource_digest_suffixer
+
+_EXPECTED_QUERY_FOR_MY_CONTENT = "_file_hash=1d079033"
+
+
+class ResourceDigestSuffixerTest(tb_test.TestCase):
+    def create_file(self, file_basename, template):
+        temp_dir = self.get_temp_dir()
+        full_path = os.path.join(temp_dir, file_basename)
+        with open(full_path, "w") as f:
+            f.write(template)
+
+        return full_path
+
+    def tearDown(self):
+        try:
+            shutil.rmtree(self.get_temp_dir())
+        except OSError:
+            # Some tests delete the directory.
+            pass
```

Comment:
We can remove this, I think? The point of `get_temp_dir()` is that it’s
automatically cleaned up.